setup.py

```diff
@@ -1,72 +1,33 @@
+from setuptools import setup, find_packages
+from setuptools.command.install import install as _install
+
 import os
 import sys
-import distutils.log
-
-from setuptools import setup, find_packages, Command
-from setuptools.command.install import install as _install
-from jinja2 import Environment, PackageLoader
-from Cython.Build import cythonize
 
 
 VERSION = "0.5.5"
 HERE = os.path.dirname(os.path.abspath(__file__))
 
 
-class GenerateList(Command):
-
-    description = "generate pyrobuf_list pxd and pyd (for development)"
-    user_options = []
-
-    def initialize_options(self):
-        self.cwd = None
-
-    def finalize_options(self):
-        self.cwd = os.getcwd()
-
-    def run(self):
-        assert os.getcwd() == self.cwd, 'Must be in package root: %s' % self.cwd
-        self.execute(_pre_install, (self, ), msg="Running pre install task")
-
-class ListAndUtil(Command):
-
-    description = "compile and install pyrobuf_list and pyrobuf_util (for development)"
-    user_options = []
-
-    def initialize_options(self):
-        self.cwd = None
-
-    def finalize_options(self):
-        self.cwd = os.getcwd()
-
-    def run(self):
-        assert os.getcwd() == self.cwd, 'Must be in package root: %s' % self.cwd
-        self.execute(_post_install, (self, ), msg="Running post install task")
-
 class install(_install):
-
     def run(self):
-        self.execute(_pre_install, (self, ), msg="Running pre install task")
-        _install.run(self)
-        self.execute(_post_install, (self, ), msg="Running post install task")
+        # By now the setup_requires deps have been fetched.
+        self.distribution.ext_modules = self.pyrobufize_builtins()
+        super(_install, self).run()
```

Comment:
No worries we should use tox to run the tests against both 2.7 and 3.5.

Thanks for your contribution a significant improvement on my initial packaging attempt!